### PR TITLE
updated a few games to working status

### PR DESCRIPTION
**Pull request by liamdawe**
StarDrive 2 works fine, Void Expanse works (requires mono), BossConstructor works, Gratuitous Space Battles 2 works, Fallen: A2P Protocol works

### CODE REVIEW

File: GAMES.json [hunk 1]
```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260": true,
```
Comment (xPaw) [inline GAMES.json]: you forgot to remove a comma here

Comment (liamdawe) [inline GAMES.json]: err sorry, haven't quite got the hang of this yet

Comment (xPaw) [inline GAMES.json]: yea my bad, true should be gone too.

Take a look at examples in readme: https://github.com/SteamDatabase/SteamLinux#contribution

**Comment by liamdawe** [comment]
Looks like i finally stoped f-ing it up :D

File: GAMES.json [hunk 2]
```diff
@@ -1183,12 +1184,17 @@
 	"323720": true,
 	"324070": true,
 	"324120": true,
+	"324260":
+	{
+		"Comment": "Requires Mono"
```
Comment (johndrinkwater) [inline GAMES.json]: This isn’t strictly something we cover in this db ‐ https://github.com/SteamDatabase/SteamLinux#contribution

Comment (directhex) [inline GAMES.json]: Has a bug been filed with the developer? Anything not in the Runtime is supposed to be bundled, distro libs/runtimes shouldn't be used

Comment (liamdawe) [inline GAMES.json]: Well, people have to install it manually @johndrinkwater, so why shouldn't it be noted?

@directhex yeah, they know about it, they haven't done anything about it

Comment (johndrinkwater) [inline GAMES.json]: What @directhex said, as well as we do not wish to collect that information. It is all in the contribution section of the README, what will be accepted and what wont.

**Comment by johndrinkwater** [comment]
@liamdawe I know the above comments look like they’re a needless blockade on submissions, but they’re not. We have a tool in the repo for linting commits (to prevent those early required edits), tool usage is mentioned in the README. We have notes to prevent addition information from getting in, as then we have to maintain that in the future, and find people to verify it if and when it changes.
If you’re happy with me using your submission, I can submit adjusted changes later when I get time.

**Comment by liamdawe** [comment]
Go ahead and use it, i will check out the tool and see.

**Pull request closed without merging**